src/mono/wasm/runtime/invoke-cs.ts

```diff
@@ -134,6 +173,14 @@ function bind_fn_0V(closure: BindingClosure) {
         try {
             const args = alloc_stack_frame(2);
             // call C# side
+            if (NativeAOT) {
```

Comment:
we have multiple copies of this code. Could we introduce `invoke_method_and_handle_exception_naot` ?